src/Utility/OrganizationFeatures.php

```diff
@@ -34,6 +34,18 @@ public static function isCpsEnabled(OrganizationInterface $org): bool
         return static::isFeatureEnabled('features.isCpsEnabled', $org);
     }
 
+    /**
+     * Checks whether pagination is enabled on an organization.
+     *
+     * @param \Apigee\Edge\Api\Management\Entity\OrganizationInterface $org
+     *
+     * @return bool
+     */
+    public static function isPaginationEnabled(OrganizationInterface $org): bool
```

Comment:
Isn't it rather `isPaginationAvailable()`? It is not an enabled feature of an organization in this sense, the result is calculated from multiple enabled features as I can see.

Reply:
Agreed, I've renamed the method.

Reply:
Good call, makes things easier to read w/proper names.